lib/recurly/shipping_address.php

```diff
@@ -2,6 +2,14 @@
 
 class Recurly_ShippingAddress extends Recurly_Resource
 {
+  public static function get($accountCode, $shippingAddressId, $client = null) {
```

Comment:
This method is already implemented on `Recurly_ShippingAddressList`. I'm okay to consider having it exposed here, but can you use the List's implementation?

https://github.com/recurly/recurly-client-php/blob/eeb0d38c153d9b25d69683eea0bae7add9873327/lib/recurly/shipping_address_list.php#L5